interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+                                                   onkeyup='datekeyup(this, mypcc)' onblur='dateblur(this, mypcc)'
+                                                   title='yyyy-mm-dd'> <img src='../pic/show_calendar.gif'
+                                                   align='absbottom' width='24' height='22' id='img_from_date'
+                                                   border='0' alt='[?]' style='cursor: pointer'
+                                                   title='<?php echo xlt('Click here to choose a date'); ?>'>
+                                        </td>
+                                        <td class='label'><?php echo xlt('To'); ?>:</td>
+                                        <td>
+                                            <input type='text' name='form_to_date' id="form_to_date"
+                                                   size='10' value='<?php echo attr($to_date) ?>'
+                                                   onkeyup='datekeyup(this, mypcc)' onblur='dateblur(this, mypcc)'
+                                                   title='yyyy-mm-dd'> <img src='../pic/show_calendar.gif'
+                                                   align='absbottom' width='24' height='22' id='img_to_date'
+                                                   border='0' alt='[?]' style='cursor: pointer'
+                                                   title='<?php echo xlt('Click here to choose a date'); ?>'>
+                                        </td>
+                                </table>
+                            </div>
+                        </td>
+                        <td class='label'><?php echo xlt('Provider'); ?>:</td>
+                        <td>
+                            <?php
+                            generate_form_field(array('data_type' => 10, 'field_id' => 'provider',
+                                'empty_title' => '-- All Providers --'), $selectedProvider);
+                            ?>
+                        </td>
+                        <td align='left' valign='middle' height="100%">
+                            <table style='border-left: 1px solid; width: 100%; height: 100%'>
+                                <tr>
+                                    <td>
+                                        <div style='margin-left: 15px'>
+                                            <a href='#' class='css_button' onclick='return submitForm();'>
+                                                <span> <?php echo xlt('Submit'); ?> </span>
+                                            </a> 
+                                            <a href='' class="css_button" id='new0' onClick=" return top.window.parent.left_nav.loadFrame2('new0', 'RTop', 'reports/daily_summary_report.php')">
+                                                <span><?php echo htmlspecialchars(xlt('Reset'), ENT_QUOTES); ?></span>
```

Comment:
do this instead (note that xlt is shorthand functions for htmlspecialchars(xl(value),ENT_NOQUOTES) ):
``````
<span><?php echo xlt('Reset'); ?></span>